GOVERNANCE.md

```diff
@@ -1,4 +1,6 @@
-# Jujutsu Governance
+---
+title: Jujutsu Governance
+---
```

Comment:
Is this a markdown table? It looks like we're sacrificing readability in the GitHub-rendered form for better rendering with starlight. I suppose there should be little reason to look at GitHub's rendering, especially if we can get renderings from PRs with starlight, so we can review the rendered output. Oh, perhaps these new titles will not be rendered as a heading at the top of the page so this is more like a directive to starlight?

Reply:
I think preserving the title in GH's rendering is worth it, since it simplifies contributions.

> Oh, perhaps these new titles will not be rendered as a heading at the top of the page so this is more like a directive to starlight?

Yes, that's the main reason.

Reply:
It is YAML frontmatter, yes. This is super common for these sorts of tools.

It renders on GitHub like this: https://github.com/steveklabnik/jj/blob/307b486f758800c9b286152a2f9189acdbc5fbe1/docs/design/git-submodule-storage.md

> Oh, perhaps these new titles will not be rendered as a heading at the top of the page so this is more like a directive to starlight?

It's a directive to Starlight but Starlight uses that directive to render the h1 with the title.

Reply:
> Anyway, I think your suggestion doesn't even work. If one file has a markdown `<h1>` tag and another file has a frontmatter plus include directive of the first file, the rendered site will end up with two identical `<h1>` tags.

It works because you can strip the title with the include syntax, just like we already do for the changelog.